app/javascript/packs/search.js

```diff
@@ -0,0 +1,34 @@
+$( function () {
+  // Set checked fields from url
+  checkboxesFromUrl("fields")
+
+  // Set checked agencies from url
+  checkboxesFromUrl("agencies")
+
+  // Add .agency-separator to agency pipe separator
```

Comment:
This is where we apply the .agency-separator class with js.

Reply:
nice!